Sources/DateScrubber.swift

```diff
@@ -14,23 +14,34 @@ public extension DateScrubberDelegate where Self: UICollectionViewController {
 }
 
 public class DateScrubber: UIViewController {
+
+    enum State {
+        case Inactive
+        case Active
+    }
+
```

Comment:
Yes, it's not very clear from a first glance what this means exactly. That it's being shown or that it's available?